### PR TITLE
NMS-13260 expand <PAGE> docs for PSM

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [X] Have you made a comment in that issue which points back to this PR?
* [X] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13260

### CODE REVIEW

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 1]
```diff
@@ -53,42 +53,42 @@ Additionally, it is possible to configure a page sequence containing a path thro
...
 
-.Page Sequence Parameter
+.Page Parameter
 [options="header, autowidth"]
 |===
 | Parameter                  | Description                                                                                | Required | Default
-| `name`                     | The name of the page-sequence. *(Is this relevant/used?)*                                  | optional | `-`
-| `method`                   | _HTTP_ method for example _GET_ or _POST_                                                  | `-`      | `-`
+| `name`                     | The name of the page-sequence.                                   | optional | `-`
+| `method`                   | _HTTP_ method for example _GET_ or _POST_                                                  | optional      | `GET`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:61]:
```suggestion
| `method`                   | _HTTP_ method. For example, _GET_ or _POST_                                                  | optional      | `GET`
```

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 2]
```diff
@@ -53,42 +53,42 @@ Additionally, it is possible to configure a page sequence containing a path thro
...
 This monitor implements the <<service-assurance/monitors/introduction.adoc#ga-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
 
-.Page Sequence Parameter
+.Page Parameter
 [options="header, autowidth"]
 |===
 | Parameter                  | Description                                                                                | Required | Default
-| `name`                     | The name of the page-sequence. *(Is this relevant/used?)*                                  | optional | `-`
-| `method`                   | _HTTP_ method for example _GET_ or _POST_                                                  | `-`      | `-`
+| `name`                     | The name of the page-sequence.                                   | optional | `-`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:60]:
```suggestion
| `name`                     | The name of the page sequence.                                   | optional | `-`
```

Comment (Bonrob2) [reply docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:60]: Unless this is modifying something?

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 3]
```diff
@@ -53,42 +53,42 @@ Additionally, it is possible to configure a page sequence containing a path thro
...
-                               (See: link:https://www.ietf.org/rfc/rfc3546.txt[RFC3546] section 3.1)                      | `-`      | `-`
-| `path`                     | The relative URL to call in the request.                                                   | required | `-`
+                               (See: link:https://www.ietf.org/rfc/rfc3546.txt[RFC3546] section 3.1)                      | optional      | `-`
+| `path`                     | The relative URL to call in the request. Any query string content after a `?` should be part of the `query` parameter.                                      | required | `-`
 | `scheme`                   | Define the _URL_ scheme as `http` or `https`                                               | optional | `http`
-| `user-info`                | Set _user info_ field in the _HTTP_ header                                                 | `-`      | `-`
+| `user-info`                | Set _user info_ field in the _HTTP_ header.  Colon separated credentials such as `USERNAME:PASSWORD`                                                 | optional      | `-`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:70]:
```suggestion
| `user-info`                | Set _user info_ field in the _HTTP_ header. Colon-separated credentials such as `USERNAME:PASSWORD`                                                 | optional      | `-`
```

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 4]
```diff
@@ -53,42 +53,42 @@ Additionally, it is possible to configure a page sequence containing a path thro
...
+| `requireIPv6`              | Communication requires a connection to an _IPv6_ address. (`true` or `false`)              | optional      | `-`
+| `requireIPv4`              | Communication requires a connection to an _IPv4_ address. (`true` or `false`)              | optional      | `-`
 | `disable-ssl-verification` | Enable or disable _SSL_ certificate verification for _HTTPS_ tests.
                                Please use this option carefully, for self-signed certificates import the CA certificate
                                in the JVM and don't just disable it.                                                      | optional | `false`
-| `port`                     | Port of the web server connecting to                                                       | optional | `80`
-| `query`                    | ??                                                                                         | `-`      | `-`
+| `port`                     | Host port number to use when connecting                                                       | optional | `80`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:77]:
```suggestion
| `port`                     | Host port number to use when connecting.                                                       | optional | `80`
```

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 5]
```diff
@@ -53,42 +53,42 @@ Additionally, it is possible to configure a page sequence containing a path thro
...
+| `query`                    | Value to concatenate with a `?` after the _path_ parameter                                                                                         | optional      | `-`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:78]:
```suggestion
| `query`                    | Value to concatenate with a `?` after the _path_ parameter.                                                                                         | optional      | `-`
```

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 6]
```diff
@@ -53,42 +53,42 @@ Additionally, it is possible to configure a page sequence containing a path thro
...
 | `failureMatch`             | Text to look for in the response body.
                                This is a _Regular Expression_ matched against every line, and it will be considered a
-                               failure at the first match and sets the service with this monitor _Down_.                  | `-`      | `-`
+                               failure at the first match and sets the service with this monitor _Down_.                  | optional      | `-`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:81]:
```suggestion
                               failure at the first match, and sets the service with this monitor _Down_.                  | optional      | `-`
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Just a few changes.

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc (hunk 2 above)
Comment (mmahacek) [reply docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:60]: This isn't really modifying anything - just removing the "is this relevant" comment.

File: docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 7]
```diff
@@ -57,38 +57,38 @@ This monitor implements the <<service-assurance/monitors/introduction.adoc#ga-se
...
-| `scheme`                   | Define the _URL_ scheme as `http` or `https`                                               | optional | `http`
-| `user-info`                | Set _user info_ field in the _HTTP_ header. Colon-separated credentials such as `USERNAME:PASSWORD`                                                 | optional      | `-`
-| `host`                     | Set _host_ field in _HTTP_ header                                                          | optional | `IP interface address of the service`
-| `requireIPv6`              | Communication requires a connection to an _IPv6_ address. (`true` or `false`)              | optional      | `-`
-| `requireIPv4`              | Communication requires a connection to an _IPv4_ address. (`true` or `false`)              | optional      | `-`
+                               negotiation, known as server name indication (_SNI_).
+                               (See: link:https://www.ietf.org/rfc/rfc3546.txt[RFC3546] section 3.1)                      | optional | `-`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc:67]:
```suggestion
                               (See: link:https://www.ietf.org/rfc/rfc3546.txt[RFC3546] section 3.1.)                      | optional | `-`
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Eventually I'll be going through everything to standardize against NH's style guide, but I did notice this period was missing.

**Pull request merged**